test/test_torch.py

```diff
@@ -2830,6 +2830,19 @@ def test_qtensor_dequantize_linear(self):
         zero_point = 2
         qt = torch._dequantize_linear(t, scale, zero_point, torch.float)
 
+    def test_qtensor_load_save(self):
+        scale = 2.0
+        zero_point = 10
+        r = torch.ones(15, dtype=torch.float) * 2
+        for dtype in [torch.quint8, torch.qint8, torch.qint32]:
+            qr = torch.quantize_linear(r, scale, zero_point, dtype)
+            # Serializing and Deserializing Tensor
+            handle = 'tensor.pt'
```

Comment:
either do save_to_buffer or create a temporary file (check what the rest of the tests do)